rclcpp/src/rclcpp/clock.cpp

```diff
@@ -52,7 +52,7 @@ Clock::now()
 {
   Time now(0, 0, rcl_clock_.type);
 
-  auto ret = rcl_clock_get_now(&rcl_clock_, &now.rcl_time_.nanoseconds);
+  auto ret = get_now_as_time_point_for_logging(&now.rcl_time_.nanoseconds);
```

Comment:
Why did this change?

Reply:
Dirk's suggestion. https://github.com/ros2/rclcpp/pull/879#discussion_r333060831

Reply:
There's not much code being duplicated... but ok I suppose.